src/main.c

```diff
@@ -663,6 +659,43 @@ poe_stream_open(char *dev, struct ustream_fd *s, speed_t speed)
 	return 0;
 }
 
+static int poet_setup(const struct port_config *ports, size_t num_ports)
```

Comment:
Happy typo? :)

Reply:
Once you find the [poem.py parser](https://github.com/Hurricos/realtek-poe/blob/mrnuke-devel/util/poem.py) you begin to realize that this is quite intentional. This is the poet that writes the serial data poem.